src/emulator/functionsEmulator.ts

```diff
@@ -860,7 +996,7 @@ export class FunctionsEmulator implements EmulatorInstance {
     }
 
     const childProcess = spawn(opts.nodeBinary, args, {
-      env: { node: opts.nodeBinary, ...opts.env, ...process.env },
+      env: { node: opts.nodeBinary, ...process.env, ...(runtimeEnv ?? {}) },
```

Comment:
FYI you can `...undefined` in a map comprehension safely.

Reply:
TIL